docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -42,6 +42,7 @@ When compiling code in any of the situations in the below table, add the corresp
 | for aarch64          | `-mbranch-protection=standard` |
 | for production code  | `-fno-delete-null-pointer-checks -fno-strict-overflow -fno-strict-aliasing -ftrivial-auto-var-init=zero` |
 | for disabling obsolete C constructs | `-Werror=implicit -Werror=incompatible-pointer-types -Werror=int-conversion` |
+| No right-to-left text in source & using GCC | `-Wbidi-chars=any` |
```

Comment:
Start with "using GCC and only left-to-right text in source code"

Reply:
Yes, done.